src/nvim/options.lua

```diff
@@ -745,6 +745,14 @@ return {
       varname='p_exrc',
       defaults={if_true={vi=false}}
     },
+    {
+      full_name='filechangenotify', abbreviation='fcnotify',
+      type='string', scope={'buffer'},
```

Comment:
it may be more forward compatible to add `list="comma"` (like the `belloff` option) so that one can add options later on ("showdiff", "autoreadfallback" etc).

Reply:
Done!